### ISSUE TITLE
Problem shown  when importing regex, Code works fine

### ISSUE THREAD

**Issue by Lulonaut** [microsoft/pylance-release#369]
## Environment data

-   Language Server version: v2020.9.4
-   OS and version: Linux, Ubuntu 20.04
-   Python version : Python 3.8.2

## Expected behaviour
No error

## Actual behaviour

"Problem" shown in Editor, code runs perfectly fine without any errors or warnings

## Logs
wont include as there is no stacktrace

## Code Snippet / Additional information

![image](https://user-images.githubusercontent.com/67191924/93108375-01eb5400-f6b3-11ea-802c-24a1f936cea9.png)

to copy:
```
An internal error occurred while type checking file "/home/lulonaut/Development/verify-private/Bot.py": TypeError: Cannot read property 'parameters' of undefined
    at /home/lulonaut/.vscode/extensions/ms-python.vscode-pylance-2020.9.4/dist/pyright.bundle.js:1:78406
    at Array.forEach (<anonymous>)
    at C.visitFunction (/home/lulonaut/.vscode/extensions/ms-python.vscode-pylance-2020.9.4/dist/pyright.bundle.js:1:78328)
    at C.visitNode (/home/lulonaut/.vscode/extensions/ms-python.vscode-pylance-2020.9.4/dist/pyright.bundle.js:1:155122)
    at C.walk (/home/lulonaut/.vscode/extensions/ms-python.vscode-pylance-2020.9.4/dist/pyright.bundle.js:1:152826)
    at /home/lulonaut/.vscode/extensions/ms-python.vscode-pylance-2020.9.4/dist/pyright.bundle.js:1:77647
    at Object.zt [as suppressDiagnostics] (/home/lulonaut/.vscode/extensions/ms-python.vscode-pylance-2020.9.4/dist/pyright.bundle.js:1:366621)
    at C.walk (/home/lulonaut/.vscode/extensions/ms-python.vscode-pylance-2020.9.4/dist/pyright.bundle.js:1:77616)
    at C._walkStatementsAndReportUnreachable (/home/lulonaut/.vscode/extensions/ms-python.vscode-pylance-2020.9.4/dist/pyright.bundle.js:1:88989)
    at C.visitSuite (/home/lulonaut/.vscode/extensions/ms-python.vscode-pylance-2020.9.4/dist/pyright.bundle.js:1:77697)Pylance

```

Note: The part were its actually used doesnt show any Problems

**Comment by erictraut** [comment]
Thanks for reporting this! Could you please provide a minimal code example that causes this problem? I'm not able to repro with just `import re`, so I presume that there's some additional context required.

**Comment by Lulonaut** [comment]
The Full code can be found [here](https://github.com/Lulonaut/Hypixel-Verify/blob/master/Bot.py)

and the part were regex is used is here(row 155-158):
```
        s = message.author.permissions_in(message.channel)
        s = str(s)
        s = re.sub("\D", "", s)
        s = int(s)

```

**Comment by Lulonaut** [comment]
Sorry for the double post but 
`s = message.author.permissions_in(message.channel)`
returns 
`<Permissions value=2081422591>` (Number might change), 

if this is helpful

**Comment by erictraut** [comment]
I cloned your repo, and I'm still not able to repro the error you're seeing. Do you have any local changes that are not yet pushed to the repo?

**Comment by erictraut** [comment]
Never mind, I just found the repro. Debugging now.

**Comment by Lulonaut** [comment]
I forgot something important i think (sorry!):

I found the source of the error but i still dont know why its there.
In row 149 is a return statement which i remove for testing. when its there the Error comes, as soon as i comment it out for testing the error goes away. Sorry, i didn't notice this before

**Comment by erictraut** [comment]
Thanks for the additional info. I found and fixed the problem. The fix will be in the next release of Pylance.

**Comment by Lulonaut** [comment]
Glad i could help and thanks for fixing it!

**Comment by jakebailey** [comment]
We typically leave the issues open until we have the fixed pushed. Reopening until then.

**Comment by Lulonaut** [comment]
Okay sorry, didnt read this anywhere

**Comment by jakebailey** [comment]
You're not expected to, it's just our convention. 🙂 

Otherwise, if we close issues when we fix them but they aren't released, others may not find the issue as easily, and the close on release provides a notification to anyone following the thread that the fix is now available.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.9.5, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202095-16-september-2020
